docs/cdn-iframe.html

```diff
@@ -8,7 +8,7 @@
 
         <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.4.4/d3.min.js"></script>
         <!-- CDN is still a WIP, when it's ready the js cdn bundle should be inserted beow! -->
-        <script src="https://cdn.jsdelivr.net/npm/britecharts@2/dist/bundled/britecharts.min.js" type="text/javascript"></script>
+        <script src="https://cdn.jsdelivr.net/npm/britecharts@1/dist/bundled/britecharts.min.js" type="text/javascript"></script>
```

Comment:
whats going on with our path here? @1 vs @2?

Reply:
Oh, this is wrong! Good catch!